config/default.json

```diff
@@ -59,6 +59,7 @@
     "dataset": {
       "title": "sandbox",
       "versionsRepositoryURL": "https://github.com/OpenTermsArchive/sandbox",
+      "versionsRepositoryURLGitLab": "https://gitlab.com/p2b/contrib-versions",
```

Comment:
The account [`p2b` on GitLab](https://gitlab.com/p2b/) is reserved. Do you own it? If not, it does not seem appropriate to send notifications to that account by default.

Reply:
The url was only used as an example, but it was our mistake since we did not realize it was pointing to an existing account. This was removed, and replaced with an obvious fake url as an example. It will be available when the pull request will be updated.